packages/nouns-subgraph/src/nouns-dao.ts

```diff
@@ -100,6 +109,35 @@ export function handleProposalCreatedWithRequirements(
   proposal.save();
 }
 
+export function handleProposalUpdated(event: ProposalUpdated): void {
+  const updateId = event.params.id
+    .toString()
+    .concat('-')
+    .concat(event.transaction.hash.toHexString());
+
+  const proposalUpdate = getOrCreateProposalUpdate(updateId);
+  const proposal = getOrCreateProposal(event.params.id.toString());
+
+  // First save the current state of the proposal to the updates history entity
+  proposalUpdate.createdAt = event.block.timestamp;
```

Comment:
@eladmallel I think `ProposalUpdate` should point to `Proposal`, i.e have a field:
`proposal: Proposal!`

Reply:
of course, total oopsie on my end :) fixed!